src/Microsoft.Health.Fhir.CosmosDb/Features/Storage/FhirDocumentClient.cs

```diff
@@ -13,22 +13,26 @@
 using Microsoft.Health.Fhir.Core.Features.Context;
 using Microsoft.Health.Fhir.Core.Features.Persistence;
 
-namespace Microsoft.Health.Fhir.CosmosDb.Features.Consistency
+namespace Microsoft.Health.Fhir.CosmosDb.Features.Storage
 {
     /// <summary>
-    /// A <see cref="IDocumentClient"/> wrapper that sets the consistency level and, if applicable, the session token
-    /// for each request.
+    /// A <see cref="IDocumentClient"/> wrapper that, for each request:
+    /// (1) sets the consistency level and, if applicable, the session token
+    /// (2) Sets <see cref="FeedOptions.ResponseContinuationTokenLimitInKb"/>
+    /// (3) Sets the <see cref="CosmosDbHeaders.RequestCharge"/> response header.
     /// </summary>
-    internal sealed partial class DocumentClientWithConsistencyLevelFromContext
+    internal sealed partial class FhirDocumentClient
```

Comment:
What about the request charged when SDK retry option is not set to 0? When the SDK retries, does it return accumulated request charges or just last operation?